pkg/backup/backup/backup_manager.go

```diff
@@ -355,6 +355,20 @@ func (bm *backupManager) makeBackupJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 		Value: string(rune(1)),
 	})
 
+	// set env vars specified in backup.Spec.Env if not present
+	for _, backupEnv := range backup.Spec.Env {
```

Comment:
What about using the function here https://github.com/pingcap/tidb-operator/blob/master/pkg/util/util.go#L256?